Ginger/GingerCoreNET/Reports/GingerReport.cs

```diff
@@ -301,7 +323,15 @@ public int TotalBusinessFlowsPassed
         {
             get
             {
-                int count = (from x in BusinessFlowReports where x.IsPassed == true select x).Count();
+                int count = 0;
+                if (BusinessFlowReports != null && BusinessFlowReports.Count > 0)
+                {
+                    count = (from x in BusinessFlowReports where x.IsPassed == true select x).Count();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5700269)